docs/source/developer/architecture/backend/algorithms/crystallographic_info_algorithm.rst

```diff
@@ -0,0 +1,43 @@
+Algorithm: `CrystallographicInfoAlgorithm`
+==========================================
+
+Description:
+------------
+This algorithm ingests crystallographic information from a CIF file or a provided
+crystallography JSON string. It generates unique reflections within a specified
```

Comment:
[`Crystallography`](https://github.com/neutrons/SNAPRed/blob/next/src/snapred/backend/dao/state/CalibrantSample/Crystallography.py) is a SNAPRed DAO.  I would also mention that this parses both in a mantid [`CrystalStructure`](https://docs.mantidproject.org/nightly/concepts/CrystalStructureAndReflections.html) object to extract the reflections.